org.eclipse.winery.frontends/app/tosca-management/src/app/instance/admin/configuration/configuration.component.ts

```diff
@@ -27,14 +28,59 @@ import { WineryNotificationService } from '../../../wineryNotificationModule/win
 
 export class FeatureConfigurationComponent implements OnInit {
     config: WineryConfiguration;
-
+    public containerUrlAvailable: boolean;
+    public containerUrlControl: FormControl = new FormControl();
+    public containerUrl;
     constructor(private http: HttpClient,
                 private configData: WineryRepositoryConfigurationService,
                 private notify: WineryNotificationService) {
     }
 
     ngOnInit(): void {
         this.config = this.configData.configuration;
+        // example: http://localhost:4242
+        // positionOfFirstColon:4, positionOfSecondColon:16
+        const positionOfFirstColon = backendBaseURL.indexOf(':');
+        const positionOfSecondColon = backendBaseURL.indexOf(':', positionOfFirstColon + 1);
+        const url = backendBaseURL.substring(0, positionOfSecondColon + 1) + '4242';
+        this.containerUrlControl.setValue(url);
+        this.containerUrl = url;
+        this.checkModelerAvailability();
+    }
+
+    checkModelerAvailability() {
+        const containerURL = (<HTMLInputElement> document.getElementById('containerUrl2')).value;
+        if (containerURL) {
+            this.containerUrl = containerURL;
+            this.http.get(this.containerUrl, { responseType: 'text' }).subscribe(
+                () => this.containerUrlAvailable = true,
+                () => this.containerUrlAvailable = false
+            );
+        }
+        if (this.containerUrl) {
+            this.http.get(this.containerUrl, { responseType: 'text' }).subscribe(
+                () => this.containerUrlAvailable = true,
+                () => this.containerUrlAvailable = false
+            );
+        }
+    }
+
+    saveModelerURL() {
+        if (this.containerUrlAvailable) {
```

Comment:
Why don't you check the availability first? Maybe the check has not returend yet, even though it is availalbe, and thus an error is displayed.
But is okay for me now. Just keep it in mind.